cogboard-app/src/main/kotlin/com/cognifide/cogboard/http/HttpClient.kt

```diff
@@ -86,18 +86,25 @@ class HttpClient : AbstractVerticle() {
     private fun initRequest(request: HttpRequest<Buffer>, config: JsonObject): HttpRequest<Buffer> {
         val user = config.getString(CogboardConstants.PROP_USER) ?: ""
         val pass = config.getString(CogboardConstants.PROP_PASSWORD) ?: ""
+        val token = config.getString(CogboardConstants.PROP_TOKEN) ?: ""
         val headers = config.getJsonObject(CogboardConstants.PROP_HEADERS)
 
-        if (user.isNotBlank() && pass.isNotBlank()) {
+        if (user.isNotBlank() && token.isNotBlank()) {
+            request.basicAuthentication(user, token)
```

Comment:
is this still `basicAuthentication` when we use token?

Reply:
yes, in this case basicAuth uses username/token instead of username/password